File: java/searching/linear_search.java
public class linear_search{

    static int linearSearch(int[] arr1, int key){
    for(int i=0;i<arr1.length;i++)
    {
        if(arr1[i]==key)
        {
            return arr1[i];
        }
    }
    return -1;
    }

    public static void main(String args[])
    {
        int arr1[] = {10,8,12,98,67};
        int key = 12;
        int data = linearSearch(arr1, key);
        if(data==-1)
        {
            System.out.println("Not found "+ key);
        }
        else{
            System.out.println("Found "+ key);
        }
    }
}

File: javascript/searching/linear_search.js
function linearSearch(arr1,key){
    for(i=0;i<arr1.length;i++)
    {
        if(arr1[i]==key)
        {
            return arr1[i];
        }
    }
    return -1;
}
var arr1 = [7,68,54,45,10];
let key = 68;
let data = linearSearch(arr1,key);
if(data==-1)
{
    console.log(`Not found ${key}`);
}
else{
    console.log(`Found ${key}`);
}

// Time Complexity:  O(N)